src/main/java/org/jenkins_ci/plugins/flexible_publish/builder/FailAtEndBuilder.java

```diff
@@ -50,8 +50,6 @@
 import hudson.tasks.Builder;
 
 /**
- * Used with {@link BuildStepRunner}.

```

Comment:
This means `org.jenkins_ci.plugins.run_condition.BuildStepRunner`.
And I believe this comment useful as this class is used only with `BuildStepRunner`.